### ISSUE TITLE
RTL UI bugs

### ISSUE THREAD

**Issue by AviFix** [BennyKok/OpenLauncher#383]
Hi,

Thanks for this app.
seems to be very good.

When running on a phone with RTL language there are a few issues with padding and directions.

![02](https://user-images.githubusercontent.com/3835504/39035838-24f18476-4484-11e8-8da4-122a857bb321.jpg)
![03](https://user-images.githubusercontent.com/3835504/39035843-269078dc-4484-11e8-89f5-aaac28d763fd.jpg)
![01](https://user-images.githubusercontent.com/3835504/39035845-287cf68e-4484-11e8-8f99-f4dbbf07b087.jpg)

Thanks

**Comment by AviFix** [comment]
Hi,
Anyone?

**Comment by dkanada** [comment]
Yeah, that definitely looks like a problem, thanks for reporting.
